test/LEAF_Request_Portal_Tests/db/seeds/BaseTestSeed.php

```diff
@@ -67,20 +67,25 @@ public function run()
             (`indicatorID`, `groupID`) VALUES
             (7, 1);
 
-            INSERT INTO `records` 
+            INSERT INTO `records`
             (`recordID`, `date`, `serviceID`, `userID`, `title`, `priority`, `lastStatus`, `submitted`, `deleted`, `isWritableUser`, `isWritableGroup`) VALUES
             (1, 1520268853, 0, 'tester', 'My Request', 0, 'Submitted', 1520268930, 0, 0, 1);
 
-            INSERT INTO `users` 
+            INSERT INTO `users`
             (`userID`, `groupID`) VALUES
             ('tester', 1),
             ('ninja', 2);
 
-            INSERT INTO `workflows` 
+            INSERT INTO `workflows`
             (`workflowID`, `initialStepID`, `description`) VALUES
             (1, 0, 'Sample Workflow'),
             (2, 0, 'Another Workflow');
 
+            INSERT INTO `workflow_steps` (`workflowID`, `stepID`, `stepTitle`, `stepBgColor`, `stepFontColor`, `stepBorder`, `jsSrc`, `posX`, `posY`, `indicatorID_for_assigned_empUID`, `indicatorID_for_assigned_groupID`, `iframeSrc`)
```

Comment:
`iframeSrc` is no longer a column in the database table. It has been replaced with `jsSrc`, which you already have in the INSERT. `iframeSrc` can be removed from the INSERT, otherwise the tests will fail.

Reply:
done